.github/workflows/docker-image.yml

```diff
@@ -5,7 +5,11 @@ on:
     branches: [ "main" ]
   pull_request: {}
   schedule:
-    - cron:  '27 4,17 * * *'
+    - cron:  '27 4,17 1-26 12 *'
+    - cron:  '27 13 27-31 12 *'
+    - cron:  '27 4,17 29,30 11 *'
+    - cron:  '27 13 1-28 11 *'
+    - cron:  '27 13 */7 1-10 *'
```

Comment:
The github web editor has tooltips for this stuff, but I'll run these through https://crontab.guru/ quick:

- At minute 27 past hour 4 and 17 on every day-of-month from 1 through 26 in December
- At 13:27 on every day-of-month from 27 through 31 in December
- At minute 27 past hour 4 and 17 on day-of-month 29 and 30 in November
- At 13:27 on every day-of-month from 1 through 28 in November
- At 13:27 on every 7th day-of-month in every month from January through October

Hope this helps.